src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/WorkflowFaultEvent.Fields.Edit.cshtml

```diff
@@ -0,0 +1,57 @@
+@using OrchardCore.Workflows.Models
+@model WorkflowFaultViewModel
+ 
+ <div class="form-group" asp-validation-class-for="ErrorFilter">
+    <label asp-for="ErrorFilter">@T["Trigger Condition"]</label>
+    <div id="@Html.IdFor(x => x.ErrorFilter)_editor" asp-for="Text" style="min-height: 400px;" class="form-control"></div>
+     <textarea asp-for="ErrorFilter" hidden>@Html.Raw(Model.ErrorFilter)</textarea>
+    <span asp-validation-for="ErrorFilter"></span>
```

Comment:
Why there's too much hints?

Reply:
This Event contains many hidden contents (this is the case with many functions of OC at present).

I hope users can directly understand how to use this event through this page
When created, the editor renders the sample usage of all properties by default.
However, if the user modifies and saves the sample code, it may be lost.
In order to ensure that the reference documents can also be directly seen during modification, I have enumerated all the attributes below

Reply:
Can we use bulleted list as hint?

Reply:
What about this?

![1660494861006](https://user-images.githubusercontent.com/15613121/184546301-02060f4a-5efa-491d-ad12-29811ca92e7b.png)

Reply:
It would be nice if we add `hint` class to the list

Reply:
updated

![image](https://user-images.githubusercontent.com/15613121/184650852-ebc1598c-ab2c-4473-bf0c-cf74c6cb9e16.png)

Reply:
I think it's better